core-web/libs/dojo-theme/scss/backend/dot-admin/portlets/_content-search.scss

```diff
@@ -24,8 +24,13 @@
     height: 100%;
 }
 
-#queryResults {
-    padding: 16px;
+.content-search__show-query-dialog {
+    .dijitDialogPaneContent {
+        padding: 0;
+    }
+    #queryResults {
+        padding: 16px;
+    }
```

Comment:
You are removing the padding in `.dijitDialogPaneContent` to add it into `#queryResults` which is a child of `dijitDialogPaneContent`, this is not need it. Why?

Reply:
Basically when you add padding apply in `.dijitDialogPaneContent ` dialog does not calculate the width properly because our content render dynamically.